docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[release-notes-header-8.4.0]]
+== 8.4
+
+[discrete]
+[[release-notes-8.4.0]]
+=== 8.4.0
+
+[discrete]
+[[known-issue-8.4.0]]
+==== Known issue
+
+
+[discrete]
+[[breaking-changes-8.4.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.4.0]]
+==== Features
+* Creates a new rule type, New Terms, that fires when a value appears for the first time in a particular field (https://github.com/elastic/kibana/pull/134526[#134526]).
```

Comment:
```suggestion
* Creates the new terms type, which generates alerts when a value appears for the first time in a particular field (https://github.com/elastic/kibana/pull/134526[#134526]).
```

Reply:
Made some edits, but @jmikell821 might have a more comprehensive summary for this one.